Rye/Rye/RyeViewController.swift

```diff
@@ -85,6 +85,7 @@ public class RyeViewController: UIViewController {
     public init(alertType: Rye.AlertType = .toast,
                 viewType: Rye.ViewType = .standard(configuration: nil),
                 at position: Rye.Position = .bottom(inset: 16),
+                animationType: Rye.AnimationType = .slideInOut,
```

Comment:
This param in init makes the api confusing since animationType can be passed in both init and configuration. I think adding an extra param `animationType: Rye.AnimationType` to the .custom viewType would be better. it could then have .slideInOut as a default param (Since swift 5.1)

Reply:
ah yeah, I agree, I'll update, thanks